crates/sui-graphql-rpc/src/types/epoch.rs

```diff
@@ -12,12 +12,12 @@ use super::transaction_block::{TransactionBlock, TransactionBlockFilter};
 use super::validator_set::ValidatorSet;
 use async_graphql::connection::Connection;
 use async_graphql::*;
-use sui_indexer::models_v2::epoch::StoredEpochInfo;
+use sui_indexer::models_v2::epoch::QueryableEpochInfo;
 use sui_types::sui_system_state::sui_system_state_summary::SuiSystemStateSummary as NativeSuiSystemStateSummary;
 
 #[derive(Clone, Debug)]
 pub(crate) struct Epoch {
-    pub stored: StoredEpochInfo,
+    pub stored: QueryableEpochInfo,
```

Comment:
Won't `validator_set` fail to compile now, because you're lacking the `system_state` field? I think you additionally need to make that field computed.

Reply:
Indeed it will!